src/main/java/core/basesyntax/model/zoo/Cat.java

```diff
@@ -1,5 +1,10 @@
 package core.basesyntax.model.zoo;
 
+import javax.persistence.DiscriminatorValue;
+import javax.persistence.Entity;
+
+@Entity
+@DiscriminatorValue("1")
```

Comment:
I would suggest to use String discriminator

Reply:
Done